boa3_test/tests/test_interop.py

```diff
@@ -1782,3 +1782,79 @@ def test_create_interop_dict(self):
     def test_verify_with_ecdsa_secp256k1_mismatched_type(self):
         path = '%s/boa3_test/test_sc/interop_test/IteratorCreateMismatchedTypes.py' % self.dirname
         self.assertCompilerLogs(MismatchedTypes, path)
+
+    def test_json_serialize(self):
+        path = '%s/boa3_test/test_sc/interop_test/JsonSerialize.py' % self.dirname
+
+        import json
```

Comment:
since this import is used in many tests, you can move this import to the top of the file